deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml

```diff
@@ -35,29 +35,24 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    categories: Integreating greatly
+    categories: RHMI integration
     certified: "false"
-    containerImage: quay.io/integreatly/integreatly-operator:v1.18.0
+    containerImage: quay.io/integreatly/integreatly-operator:v1.19.0
     createdAt: "2019-10-15 10:39:00"
-    description: Greatly integrating integreatly greatness
-    support: Integreatly
-  name: integreatly-operator.v1.18.0
+    description: RHMI integration suite of tools
+    support: RHMI
+  name: integreatly-operator.v1.19.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: RHMI
-      description: Represents an installation of RHMI.
-      displayName: RHMI Installation
```

Comment:
Just testing an olm install and looks like we need at least `displayName` in `spec.customresourcedefinitions.owned` for the InstallPlan & Subscription to resolve. 

I tested generating a new csv based off this one, and it looks like these fields no longer get brought over from the previous csv using operator-sdk 0.15.1 :confused: :man_shrugging:

Reply:
I've re-added these now